src/action/setting.js

```diff
@@ -0,0 +1,22 @@
+import { UNITS } from '../config';
+
+class SettingAction {
+  constructor(store) {
+    this._store = store;
+  }
+
+  //
+  // Bitcoin Unit actions
+  //
```

Comment:
We should add an init method here to stay consistent with other actions see e.g. InvoiceAction.

Reply:
i missed that, thanks!